src/clients/api/queries/useGetTreasuryTotals/index.ts

```diff
@@ -32,7 +25,15 @@ export interface UseGetTreasuryTotalsOutput {
 
 const useGetTreasuryTotals = (): UseGetTreasuryTotalsOutput => {
   const { accountAddress, chainId } = useAuth();
-  const treasuryAddress = TREASURY_ADDRESSES.get(chainId);
+  const treasuryAddress = useMemo(() => {
+    switch (chainId) {
+      case ChainId.BSC_MAINNET:
+      case ChainId.BSC_TESTNET:
+        return getVTreasuryContractAddress({ chainId });
+      default:
+        return getVTreasuryV8ContractAddress({ chainId });
+    }
+  }, [chainId]);
```

Comment:
`treasuryAddress` is then passed on to the Venus Lens contract, so we need to double check if it's going to be deployed to the new chains

Reply:
I've added Sepolia's Venus Lens address to move this forward. There are other contracts needed to display treasury amounts though

Reply:
So far so good, but then I'd suggest keeping this PR as draft and waiting for all the necessary contracts to be deployed to Sepolia before we review and merge it